src/AudioAnalysisTools/Events/EventFilters.cs

```diff
@@ -253,7 +259,10 @@ public static List<EventCommon> FilterEventsOnSyllableCountAndPeriodicity(List<E
                 string strArray = DataTools.Array2String(actualPeriodSeconds.ToArray());
                 Log.Debug($" Event{count} actual periods: {strArray}");
 
-                    // reject composite events whose total syllable count exceeds the user defined max.
+                // don't use following line because does not format decimal places.
+                //Log.Debug($" Event{count} actual periods: {actualPeriodSeconds.Join(",")}");
```

Comment:
then add a format string overload?

Reply:
THis has been fixed by the creation of a new extention method